source/_integrations/wemo.markdown

```diff
@@ -72,6 +72,26 @@ Note that if you use static device entries, you may want to set up your router (
 
 If the device doesn't seem to work and all you see is the state "unavailable" on your dashboard, check that your firewall doesn't block incoming requests on port 8989, since this is the port to which the WeMo devices send their updates.
 
+### Device Options
+
+Clicking the `Configure` button on the WeMo integration will bring up some additional options that can be configured for WeMo devices.
+
+![Device Options](/images/integrations/wemo/device_options.png)
+
+**Subscribe to device local push updates**: WeMo devices support both the *Local Push* and *Local Polling* [IoT classes](https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things/#classifiers). By default Home Assistant will subscribe to event notifications from WeMo devices and use the Local Push IoT class. As a fallback, if the Local Push doesn't work, Home Assistant will use Local Polling. Some devices are known not to work well with Local Push. WeMo devices expect to be on the same subnet as Home Assistant and will not work with Local Push otherwise. For devices that are known not to work with Local Push, the *Subscribe to device local push updates* option can be disabled to force only Local Polling to be used.
+
+There are some downsides of disabling *Subscribe to device local push updates*:
+
+- The WeMo Motion detector will not work in Home Assistant when *Subscribe to device local push updates* is disabled. The same will be the case for the sensor on the WeMo Maker device. Without a push subscription, Home Assistant will be unaware of motion events.
+
+- Long press events, when the button on a wall switch/dimmer is pressed, will not work if *Subscribe to device local push updates* is disabled.
+
+- Automations based on the device being locally switched on or off will be delayed by at least the polling interval (below).
+
+**Register for device long-press events**: WeMo wall switches and dimmers will send Home Assistant a notification when the button on the device is held for more than 2 seconds. This feature is enabled by default in Home Assistant (see [below](#long-press-events-and-triggers)). If this feature causes issues for your device, it can be disabled by deselecting the *Register for device long-press events* option.
```

Comment:
```suggestion
**Register for device long-press events**: WeMo wall switches and dimmers will notify Home Assistant when the button on the device is held for more than 2 seconds. This feature is enabled by default in Home Assistant (see [below](#long-press-events-and-triggers)). If this feature causes issues for your device, it can be disabled by deselecting the **Register for device long-press events** option.
```